src/main/java/Item.java

```diff
@@ -0,0 +1,31 @@
+public abstract class Item {
+    private boolean isDone = false;
+    private final String done = "[X] ";
```

Comment:
If these strings are final and constants, perhaps change the naming to be all CAPS?